business-registry-dashboard/app/locales/en-CA.ts

```diff
@@ -186,7 +186,7 @@ export default {
     LIQUIDATION: 'This business is in liquidation',
     DISSOLUTION: 'This business is in the process of being dissolved',
     PROCESSING: 'This name request is still processing, it may take up to 10 minutes.',
```

Comment:
While you're here, this is a run-on sentence. Change the comma to a period to fix it :)